src/main.js

```diff
@@ -188,10 +188,53 @@ function languageCodeIsCompatible(language, languages) {
     return languages.some(l => l === languageCode);
 }
 
+function flatpakSystemDirs() {
+  let env = GLib.getenv('EOS_DISCOVERY_FEED_FLATPAK_SYSTEM_DIRS');
+  if (env)
+      return env.split(':');
+
+  return ['/var/lib/flatpak', '/var/endless-extra/flatpak'];
+}
+
+// We need to look at the flatpak directories and /var/lib explicitly
+// since they are not included in XDG_DATA_DIRS
+function allRelevantDataDirs() {
+  return GLib.get_system_data_dirs().concat(flatpakSystemDirs().map((directory) =>
```

Comment:
/va/rlib/flatpak/exports/share system dirs doesn't exist in XDG_DATA_DIRS in the sandbox, sadly.

Reply:
I think you probably still want to ensure each directory only exist once in the array.

Reply:
Makes sense.